simdjson-flags.cmake

```diff
@@ -36,7 +36,7 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 #
 add_library(simdjson-flags INTERFACE)
 if(MSVC)
-  target_compile_options(simdjson-flags INTERFACE /nologo /D_CRT_SECURE_NO_WARNINGS)
+  target_compile_options(simdjson-flags INTERFACE /nologo)
```

Comment:
Why is /nologo even needed? Let the compiler print its version number and copyright if it wants... No?

Reply:
I can't think of any reason. I'll remove it while I'm in there.